RecoTracker/TkNavigation/plugins/SimpleNavigableLayer.cc

```diff
@@ -225,7 +225,7 @@ std::vector< const DetLayer * > SimpleNavigableLayer::compatibleLayers (const Fr
   Lvect && someLayers = nextLayers(fts,timeDirection);
   if (someLayers.empty()) {
     LogDebug("SimpleNavigableLayer")  <<"Number of compatible layers: "<< 0;
-    return someLayers;
+    return std::move(someLayers);
```

Comment:
similar comment here as in the other PR: perhaps it's better to change someLayers to not be an rvalue ref.
IIUC, here it can be ```const Lvect& someLayers```

Reply:
@smuzaffar 
please clarify if you are going to update this PR following the review comment.
Thank you.

Reply:
@slava77 , yes I will update it soon.

Reply:
fixed